ckan/controllers/user.py

```diff
@@ -395,7 +395,7 @@ def login(self, error=None):
             item.login()
 
         if 'error' in request.params:
-            h.flash_error(request.params['error'])
+            error = request.params['error']
```

Comment:
this is a change that we'll have to justify keeping as we upgrade to 2.8 and 2.9 so it's worth adding a comment here so we can have a record of the reason for the change. Something like `# pass error to template instead of with flash_error so it can be captured for GA events in our overridden templates`